frontend/modules/recipe/components/Recipe.js

```diff
@@ -126,17 +126,12 @@ var RecipeScheme = injectIntl(React.createClass({
         description: 'minutes',
         defaultMessage: 'minutes'
       },
-      photo_placeholder: {
-        id: 'recipe.photo_placeholder',
-        description: 'Photo placeholder',
-        defaultMessage: 'Generic placeholder thumbnail'
-      }
     });
 
     return (
       <div className="recipe-details">
         <h1 className="title hidden-xs">{this.props.data.title}</h1>
```

Comment:
Should we only show this header if there is an Image? Looks weird to have it twice so quickly. Thoughts?